build.gradle

```diff
@@ -14,22 +14,25 @@ buildscript {
         classpath 'com.palantir.javaformat:gradle-palantir-java-format:2.67.0'
         classpath 'com.palantir.suppressible-error-prone:gradle-suppressible-error-prone:2.9.0'
         classpath 'gradle.plugin.org.inferred:gradle-processors:3.7.0'
+        classpath 'com.palantir.baseline:gradle-baseline-java:6.5.0'
     }
 }
 
 plugins {
     id 'com.gradle.plugin-publish' version '1.3.1'
-id 'com.palantir.git-version' version '3.3.0'
+    id 'com.palantir.git-version' version '3.3.0'
     id 'com.palantir.idea-test-fix' version '0.1.0'
- id 'com.palantir.baseline' version '6.32.0'
-id 'com.palantir.consistent-versions' version '2.34.0'
+    id 'com.palantir.baseline' version '6.32.0'
```

Comment:
do we want both this and lines 17? they're conflicting versions no?

Reply:
okay, ran it through a build scan, it doesn't actually matter, it will just use `6.32.0`. But it is confusing, can we pick which one we want to use? i.e. either `apply plugin: 'com.palantir.baseline'` or apply it here in the plugins block.

Not sure how it works with `baseline-java-versions`, whether it can be applied via the plugins block too.

Reply:
thanks for noticing! I just tried to indent this block without reading it through. addressed in https://github.com/palantir/gradle-git-version/pull/972/commits/c3426bbff705921100a9da566ff4b6213735184e

Reply:
I think you'll need to bump the version in the `classpath` declaration